rpcapd/log.c

```diff
@@ -26,6 +26,7 @@
 #include <stdlib.h>
 
 #ifdef _WIN32
+#define WIN32_LEAN_AND_MEAN
```

Comment:
You should add a comment like `exclude some of the less common API declarations`.

Reply:
Will do. This was necessary because some of the appveyor builds were failing when Winsock2.h was included where windows.h had already included winsock.h.

Reply:
> This was necessary because some of the appveyor builds were failing when Winsock2.h was included where windows.h had already included winsock.h.

This should be explained in a comment.

Reply:
The comment now includes @dmiller-nmap's explanation; @fxlb, does that suffice?